### ISSUE TITLE
Table Of Contents for Page - At current location - can put in wrong place on page (rarely)

### ISSUE THREAD

**Issue by jasonjac2** [stevencohn/OneMore#1585]
# Problem to Solve
This is another expression of: #1563 
If you have multiple instances of ON running and you want to insert a ToC at the current location on the current page.  You follow the instructions and the ToC appears in the wrong place, i.e. it appears where the other instance of ON has the editor point.

You may need to do this twice, once from the 2nd instance and once from the 1st instance - I am not sure on the logic that OM uses to find the current page.
# To Reproduce

Open ON
Open another instance (ctrl+m)
On the first instance go down the page a bit (you need a large enough page) and click somewhere so the editor point is there.
Goto the other instance
Goto somewhere near the top of the page
Invoke ToC for the current page at the current location
The ToC isn't where your editor point is.

# Environment (if applicable)
Version 6.5.1 with OneNote 16.0.17928.20156

# Additional Context

**Comment by jasonjac2** [comment]
Just bumped into this again and I have cited it in #1563 
I think my conclusions are:
- if you have 2 instances of ON open on the same page and the other instance is the one that is going to report the XML to the OM tool. Then if you tell OM to insert a ToC at the current location:
- it will insert it at the 1st instances selection
- OR it will not insert anything if the first instance has the selection in the page title.

I hope that clarifies the situation.

**Comment by stevencohn** [comment]
I agree this is likely related to #1563, which means there little I can do here to help it along. I'll have to close this for now.

**Comment by jasonjac2** [comment]
... other than detect the issue and warn the user...... otherwise your kit gets blamed for ON's woes.
